app/api/views.py

```diff
@@ -345,7 +345,10 @@ def file_view(request, idname):
     """
     kwargs = id_or_name(idname)
     log.debug('kwargs: %s', kwargs)
-    file = get_object_or_404(Files, user=request.user, **kwargs)
+    if request.user.is_superuser:
```

Comment:
This whole if can just be (since you already have the kwargs dict):

```python
if not request.user.is_superuser:
	kwargs['user'] = request.user
file = get_object_or_404(Files, **kwargs)
```